src/features/søknad/utils.ts

```diff
@@ -1,9 +1,77 @@
 import { Adresse } from '~src/api/personApi';
 import { Nullable } from '~src/lib/types';
-import { EktefellePartnerSamboer } from '~src/types/Søknad';
+import { EktefellePartnerSamboer, SøknadFellesInnhold, SøknadInnholdAlder, Søknadstype } from '~src/types/Søknad';
 
-import { AdresseFraSøknad, SøknadState } from './søknad.slice';
-import { EPSFormData } from './types';
+import { AdresseFraSøknad, AlderssøknadState, SøknadFellesState, SøknadState } from './søknad.slice';
+import { DelerBoligMed, EPSFormData } from './types';
+
+export const toUføreinnsending = (søknad: SøknadState, fnr: string) => ({
+    uførevedtak: {
+        harUførevedtak: søknad.harUførevedtak!,
+    },
+    flyktningsstatus: {
+        registrertFlyktning: søknad.flyktningstatus.erFlyktning!,
+    },
+    oppholdstillatelse: {
+        erNorskStatsborger: søknad.flyktningstatus.erNorskStatsborger!,
+        harOppholdstillatelse: søknad.flyktningstatus.harOppholdstillatelse,
+        typeOppholdstillatelse: søknad.flyktningstatus.typeOppholdstillatelse,
+        statsborgerskapAndreLand: søknad.flyktningstatus.statsborgerskapAndreLand!,
+        statsborgerskapAndreLandFritekst: søknad.flyktningstatus.statsborgerskapAndreLandFritekst,
+    },
+    ...toFellessøknadsinnsending(søknad, fnr),
+});
+
+export const toAldersinnsending = (søknad: AlderssøknadState, fnr: string): SøknadInnholdAlder => ({
+    harSøktAlderspensjon: søknad.harSøktAlderspensjon ?? false,
```

Comment:
Jeg ville heller ha banget på denne. Da den strengt tatt skal være utfylt på dette tidspunktet. Hvis ikke, så vil jeg heller at den feiler på at den er null, og ikke at vi potensielt sender inn et svar for søkeren.